grammars/asciidoc.cson

```diff
@@ -48,7 +48,7 @@
   # ========================
   #
   {
-    'begin': '^\\.(\\w[\\w ]*\\w)'
+    'begin': '^\\.(\\w.*\\w)'
```

Comment:
I agree. I am just wonder why the original matches use `\w`. Are there special considerations?